app/controllers/api/discussions_controller.rb

```diff
@@ -2,19 +2,13 @@ class API::DiscussionsController < API::RestfulController
   load_and_authorize_resource only: [:show, :mark_as_read, :set_volume]
   load_resource only: [:create, :update]
 
-  def inbox_by_date
-    load_and_authorize_group if params[:group_id]
-    @discussions = page_collection inbox_threads
+  def discussions_for_dashboard
+    @discussions = page_collection discussions_for_preview
     respond_with_discussions
   end
 
-  def inbox_by_organization
-    @discussions = grouped inbox_threads.group_by(&:organization_id)
-    respond_with_discussions
-  end
-
-  def inbox_by_group
-    @discussions = grouped inbox_threads.group_by(&:group_id)
+  def discussions_for_inbox
```

Comment:
These queries are finally starting to converge, although I'm not quite ready to collapse them just yet.